doc/plugins.md

```diff
@@ -170,6 +170,9 @@ The list of plugins:
     — replace slot elements with injected content
 *   [`rehype-slug`](https://github.com/rehypejs/rehype-slug)
     — add `id`s to headings
+*   [`rehype-slug-custom-id`](https://github.com/unicorn-utterances/rehype-slug-custom-id)
+    — add `id`s to headings, with the option to pass a custom ID in the HTML
+    innerText itself
```

Comment:
Awesome!!

```suggestion
    — add `id`s to headings, also supports `{#custom-ids}`
```

How about this? I think it might be clearer (and I like that it’s shorter)

Reply:
Much better - commited